src/main/java/org/apache/sysds/runtime/compress/readers/ReaderColumnSelection.java

```diff
@@ -106,6 +106,30 @@ else if(rawBlock.getDenseBlock().numBlocks() > 1)
 			return new ReaderColumnSelectionDenseMultiBlock(rawBlock, colIndices, rl, ru);
 		return new ReaderColumnSelectionDenseSingleBlock(rawBlock, colIndices, rl, ru);
 	}
+	//TODO Add ReaderColumnSelectionDenseDeltaSingleBlock to the method
+	public static ReaderColumnSelection createDeltaReader(MatrixBlock rawBlock, IColIndex colIndices, boolean transposed,
+													 int rl, int ru) {
+		checkInput(rawBlock, colIndices, rl, ru);
```

Comment:
The content of this method is not valid, therefore throw NotImplementedException.